docs/user-guide/persistent-volumes/index.md

```diff
@@ -399,3 +399,27 @@ parameters:
 
 * `type`: [VolumeType](http://docs.openstack.org/admin-guide/dashboard-manage-volumes.html) created in Cinder. Default is empty.
 * `availability`: Availability Zone. Default is empty.
+
+## Writing Portable Configuration
+
+When writing configuration templates or examples that need persistent storage
```

Comment:
A little clearer if you reverse the order in this sentence.

"If you're writing configuration templates or examples that run on a wide range of clusters and need persistent storage, we recommend that you use the following pattern:"